src/main/java/com/iota/iri/conf/MilestoneConfig.java

```diff
@@ -6,12 +6,16 @@
 public interface MilestoneConfig extends Config {
 
     /**
-     * @return Descriptions#COORDINATOR
+     * Default Value: {@value BaseIotaConfig.Defaults#COORDINATOR_ADDRESS}
```

Comment:
the default value name does not match the description name.
`COORDINATOR_ADDRESS` vs `COORDINATOR`